examples/stencil/src/components.d.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
```

Comment:
Since this file is generated, shouldn't it be gitignore'd?

Reply:
I believe that's a would be a question best answered by the Stencil team. The gitignore that is created when you init a new stencil project doesn't ignore it. I'm not really sure if there's a reason for that.

In Stencil projects I maintain, we've always committed this file. 🤷‍♂️

Reply:
I'll keep it only because a fresh `build` will fail if the `src/components.d.ts` file isn't already present. However, it works when running `watch` from a clean slate.